demisto_sdk/commands/upload/uploader.py

```diff
@@ -130,11 +130,25 @@ def directory_uploader(self, path: str):
             path (str): Path for directory to upload.
         """
         if is_path_of_integration_directory(path):
+            # Upload unified integration files
+            list_unified_integrations = get_child_files(path)
```

Comment:
the var name is a bit misleading here, its not really a list of unified integrations right?

Reply:
It is. Could be a list of them.

Reply:
that func just returns all the files that are under the given dir path, how do you know they are all unified integrations? how can there be more than one?

Reply:
Consider this:
![demisto-sdk  ~:dev:demisto:demisto-sdk  -  :demisto_sdk:tests:test_files:Packs:DummyPack:Scripts:DummyScriptUnified2 yml 2020-06-01 16-05-59](https://user-images.githubusercontent.com/44666568/83411937-e620b280-a421-11ea-803b-521f616f93bf.png)

Both `DummyScriptUnified` and `DummyScriptUnified2` could be unified scripts under the `Scripts` directory. I don't know they are unified I can only check if they are integrations and try to upload them. Same goes for integrations.

Reply:
ok but the var name is still misleading, it is not a list of unified entities, but a list of all the files under that dir

Reply:
Which I assume to be valid unified integrations/scripts. I can change it to `files` but then IMO it is even more misleading as it is not clear why am I iterating over random files in a directory as opposed to iterating over unified integrations/scripts. What ever you think